Xamarin.Forms.Platform.Android/AppCompat/ButtonRenderer.cs

```diff
@@ -9,11 +9,14 @@
 using Android.Util;
 using GlobalResource = Android.Resource;
 using Object = Java.Lang.Object;
+using AView = Android.Views.View;
+using AMotionEvent = Android.Views.MotionEvent;
+using AMotionEventActions = Android.Views.MotionEventActions;
 using static System.String;
 
 namespace Xamarin.Forms.Platform.Android.AppCompat
 {
-	public class ButtonRenderer : ViewRenderer<Button, AppCompatButton>, global::Android.Views.View.IOnAttachStateChangeListener
+	public class ButtonRenderer : ViewRenderer<Button, AppCompatButton>, AView.IOnAttachStateChangeListener
```

Comment:
This is simply because consistency and efficiency. 
Do you want to keep "global::Android.Views.View" only here? Or, doens't want to use alias "AView" ?